test/chi/chi-vue/data-table-portal.pug

```diff
@@ -0,0 +1,683 @@
+script(src="https://cdn.jsdelivr.net/npm/vue@2.6.11")
+script(src="https://ffa501f0173abba98f9476d13fde468ee1ac8e93-chi-pr.enterprise.ctl.io/chi-vue/umd/chi-vue.umd.js")
+
+h2 Data Table Portal
+h3 Portal
+.test-data-table.-p--3.-w--100
+    #portal
```

Comment:
Indentation on this page looks a bit off. Can you make sure that it's indented using 2 spaces as it is standard in Chi?

Reply:
My formatter still doesn't work with pug files so I don't know how to apply the 2 spaces rule other than manually for now :/ but i'm gonna try to find an extension that works